modular_chomp/code/game/objects/items/weapons/circutboards/computer/research.dm

```diff
@@ -0,0 +1,7 @@
+//Public RND console for outsider pod use
+/obj/item/weapon/circuitboard/rdconsole/public
+	name = T_BOARD("Outsider R&D control console")
+	build_path = /obj/machinery/computer/rdconsole/public
+
+/obj/item/weapon/circuitboard/rdconsole/attackby(obj/item/I as obj, mob/user as mob) //It has access change interactions with screwdriver for the main board, you cant change them here.
```

Comment:
All rdconsole boards? Not only the public subtype?

Reply:
Oops, missed that, will fix soon